wp-includes/integrations/class-wp-service-worker-styles-integration.php

```diff
@@ -68,11 +76,15 @@ public function register( WP_Service_Worker_Cache_Registry $cache_registry ) {
 
 			$revision = false === $dependency->ver ? get_bloginfo( 'version' ) : $dependency->ver;
 
-			/** This filter is documented in wp-includes/class.wp-styles.php */
-			$url = apply_filters( 'style_loader_src', $url, $handle );
+			// @todo Look into why this fails with 'colors', note that $_wp_admin_css_colors is null at this moment.
+			if ( 'colors' !== $handle ) {
```

Comment:
For some reason `colors` style fails both in PHP and also in Workbox. In PHP it looks like it might be due to `$_wp_admin_css_colors` not being set when calling `style_loader_src` filter.

In Workbox the output in console is the following:
![screen shot 2018-09-11 at 5 00 48 pm](https://user-images.githubusercontent.com/3294597/45384630-ebeaba80-b5e4-11e8-975b-988e1cf64cf2.png)